server/app/services/ti/TIClientCreationResult.java

```diff
@@ -0,0 +1,34 @@
+package services.ti;
+
+import forms.AddApplicantToTrustedIntermediaryGroupForm;
+import java.util.Optional;
+import play.data.Form;
+
+public class TIClientCreationResult {
```

Comment:
suggestion (blocking): please document this class and make it `final`

Reply:
question (blocking): it looks like this class is just a wrapper around `Optional<Form<AddApplicantToTrustedIntermediaryGroupForm>>` without storing any other state or providing other functionality. Is it needed?

Reply:
Yes in future, We may add the form for updating Client DOB.. it is needed...

Reply:
So the plan is for it to wrap two separate forms in the future?

Reply:
Yes..